src/main/java/frc/robot/Constants.java

```diff
@@ -32,9 +34,27 @@ public final class Constants {
       Seconds.of(0.13); // s, 20ms + 110ms sprk max velocity lag
   public static final Measure<Velocity<Distance>> MAX_SPEED = FeetPerSecond.of(14.5);
 
-  public static final class AutoConstants {
+  public static final class PivotConstants {
+    public static final double VORTEX_ENCODER_TICKS_PER_REVOLUTION = 7168;
     public static final PIDConstants TRANSLATION_PID = new PIDConstants(0.7, 0, 0);
     public static final PIDConstants ANGLE_PID = new PIDConstants(0.4, 0, 0.01);
+    public static final double PIVOT_kP = 0;
+    public static final double PIVOT_kI = 0;
+    public static final double PIVOT_kD = 0;
+    public static final double PIVOT_kS = 0;
+    public static final double PIVOT_kG = 0;
+    public static final double PIVOT_kV = 0;
+    public static final double PIVOT_kA = 0;
+    public static final int PIVOT_MOTOR = 20;
+    public static final Measure<Velocity<Angle>> PIVOT_MAX_VEL = RPM.of(4);
+    public static final Measure<Velocity<Velocity<Angle>>> PIVOT_MAX_ACCEL =
+        MetersPerSecondPerSecond.of(20);
+    public static final Measure<Mass> PIVOT_WEIGHT = Kilogram.of(9.55);
+    public static final Measure<Velocity<Angle>> MOTOR_ZERO_SPEED = RPM.of(0);
```

Comment:
We actually use this a power not as a velocity angle so it can just be a double
```suggestion
    public static final double MOTOR_ZERO_SPEED = RPM.of(0);
```